examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -66,6 +64,38 @@ export interface ITask extends IEventProvider<ITaskEvents> {
      * The task priority.  Modifications are persisted in Fluid and shared amongst collaborators.
      */
     priority: number;
+    /**
+     * The task name coming in from the external server.
+     */
+    diffName: string;
+    /**
+     * The task priority coming in from the external server.
+     */
+    diffPriority: number;
+    /**
+     * The type of change to the task coming in from the external server.
+     */
+    diffType: string;
+    /**
+     * Trigger event to render change to UI.
+     */
+    readonly externalNameChanged:(name: string) => Promise<void>;
+    /**
+     * Trigger event to render change to UI.
+     */
+    readonly externalPriorityChanged:(priority: number) => Promise<void>;
+    /**
+     * Save the proposed changes to SavedData.
+     */
+    readonly acceptChange:() => Promise<void>;
+    /**
+     * Ignore the proposed changes to SavedData.
+     */
+    readonly ignoreChange:() => Promise<void>;
+
+    readonly DEFAULT_PRIORITY: number;
+    readonly DEFAULT_NAME: string;
+    readonly DEFAULT_DIFF_TYPE: string;
```

Comment:
Any reason these are all in caps? I'm used to seeing constants written like that so it confused me for a sec.

Reply:
I want them to behave like constants, so that's why I put them in all caps. The problem with that is where they should live and how to import them in the right places.  Let me see if I can treat them as proper constants.